packages/circuit-ui/package.json

```diff
@@ -44,7 +44,7 @@
     "react-modal": "^3.8.1",
     "react-number-format": "^4.4.1",
     "react-popper": "^2.2.5",
-    "react-use": "^17.2.4",
```

Comment:
Can you remind me why we decided to move away from a hooks library for things like useClickOutside and implement it ourselves? I thought we wanted to keep react-use for now and move to react-hookz in a 3.x version

Reply:
I tried to explain this in the PR description: 

> Its event listener can be deactivated to improve performance (e.g. when the Popover is closed). Unlike the previously used hook from `react-use` it listens for `click` events which can be prevented from bubbling using `event.stopPropagation()`. This makes it simpler to exclude a trigger element from outside clicks.

`react-hookz` has the same shortcomings as `react-use` in this case.

Reply:
Oops I missed that part–thanks, this makes sense!

Moving forward though, would we still want to move towards a hooks library, and maybe submit PRs to the third-party lib directly if we notice room for improvements?

Reply:
Yeah, I think that makes sense.